packages/aws-cdk-lib/aws-ec2/lib/vpc-endpoint.ts

```diff
@@ -660,8 +661,21 @@ export class InterfaceVpcEndpointAwsService implements IInterfaceVpcEndpointServ
         'redshift', 'redshift-data', 's3', 'sagemaker.api', 'sagemaker.featurestore-runtime', 'sagemaker.runtime', 'securityhub',
         'servicecatalog', 'sms', 'sqs', 'states', 'sts', 'sync-states', 'synthetics', 'transcribe', 'transcribestreaming', 'transfer',
         'workspaces', 'xray'],
+      'us-isof-': ['ecr.api', 'ecr.dkr'],
+      'eu-isoe-': ['ecr.api', 'ecr.dkr'],
```

Comment:
Do we only need this two here? What about other us gov regions?

Reply:
these two are the only ones flagged in ticket, so for now only these two. These are isolated, for US Gov region i think it starts with `us-gov`